src/Validator/Manager.php

```diff
@@ -19,4 +20,19 @@ class Manager extends AbstractPluginManager
      * @see AbstractPluginManager::$instanceOf
      */
     protected $instanceOf = ValidatorInterface::class;
+
+    /**
+     * Returns the validator specified in the config if found. if not return the default RequiredMembers validator
+     *
+     * @param $request
+     * @return ValidatorInterface
+     */
+    public function getValidatorByRequest(AbstractRequest $request) : ValidatorInterface {
+        $options = $request->getOptions();
+        if(isset($options['validator'])) {
```

Comment:
Due to the fact that the validator can be given through the configuration check in the manager if the class exists:
```php
if (true === isset($options['validator'])) {
    if (true === is_string($options['validator']) && true === $this->has($options['validator'])) {
        return $this->get($options['validator']);
    } elseif (false === is_callable($options['validator']) {
        throw new RuntimeException(....) // TODO: make sure you use the correct exception class
    }
    return ($options['validator'])();
}
return $this->get('RequiredMembers');
```